README.md

```diff
@@ -38,13 +38,13 @@ sudo mv winetricks /usr/local/bin
 ```
 Note: /usr/local/bin must be in your $PATH for this to work.
 
-You can then keep winetricks up to date in the future by doing:
+Winetricks can be kept updated by doing:
```

Comment:
"can be updated"

"kept" implies that it will be automatically updated in the future by that command, which isn't the case.